<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagerDialog</class>
 <widget class="QDialog" name="ManagerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DSC :: Manager Panel</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>711</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Final Income:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Overall Income:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QTableWidget" name="transactions">
      <column>
       <property name="text">
        <string>User ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Price</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Airline</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ticket No.</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Billing Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Billing Address</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Phone Number</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="txtFinalIncome">
      <property name="text">
       <string>null</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Commission:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="txtIncome">
      <property name="text">
       <string>null</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="txtCommission">
      <property name="text">
       <string>null</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
